tools/codegen/gen_backend_stubs.py

```diff
@@ -241,15 +241,25 @@ def gen_dispatcher_registrations(
         grouped_native_functions: Sequence[Union[NativeFunction, NativeFunctionsGroup]],
         backend_dispatch_key: DispatchKey,
         dispatch_key: DispatchKey,
-        selector: 'SelectiveBuilder') -> None:
+        selector: 'SelectiveBuilder',
+        build_in_tree: bool = False,
```

Comment:
Why do we need the flag in `gen_backend_stubs.py` (I thought it wasn't used by LTC?)

Also, I think a light comment here on why we have `build_in_tree` would be nice. Basically, `gen_backend_stubs.py` was originally meant as a codegen pipeline for external backends that don't have the ability to take advantage of `native_functions.yaml` and our core codegen pipeline. And now that LTC is a part of core, it may or may not get built in-tree (depending on whether the backend that hooks into LTC is in-tree or not).

Reply:
ohh it's just because `gen_lazy_tensor.py` calls into the function in `gen_backend_stubs.py` (so if you're adding a new flag for the LTC codegen, the backend_stubs function needs to know about it). I guess that's fine, since the `gen_backend_stubs.py` entrypoint isn't affected - just the helper functions it uses. (Maybe just include in the comment here that it's only actually needed/used by LTC codegen)

Reply:
I added a comment locally, but if my CI is clean this time I'm gonna land the comment in a separate diff haha